grafana-plugin/src/pages/schedule/Schedule.tsx

```diff
@@ -276,33 +391,40 @@ class _SchedulePage extends React.Component<SchedulePageProps, SchedulePageState
-                      onUpdate={this.refreshEventsAndClearPreview}
-                      onDelete={this.refreshEventsAndClearPreview}
-                      shiftIdToShowRotationForm={shiftIdToShowRotationForm}
-                      onShowRotationForm={this.handleShowRotationForm}
-                      onShowOverrideForm={this.handleShowOverridesForm}
-                      disabled={disabledRotationForm}
-                      filters={filters}
-                      onShowShiftSwapForm={!shiftSwapIdToShowForm ? this.handleShowShiftSwapForm : undefined}
-                      onSlotClick={shiftSwapIdToShowForm ? this.adjustShiftSwapForm : undefined}
-                    />
-                    <ScheduleOverrides
-                      scheduleId={scheduleId}
-                      onCreate={this.refreshEventsAndClearPreview}
-                      onUpdate={this.refreshEventsAndClearPreview}
-                      onDelete={this.refreshEventsAndClearPreview}
-                      shiftIdToShowRotationForm={shiftIdToShowOverridesForm}
-                      onShowRotationForm={this.handleShowOverridesForm}
-                      disabled={disabledOverrideForm}
-                      disableShiftSwaps={disabledShiftSwaps}
-                      shiftStartToShowOverrideForm={shiftStartToShowOverrideForm}
-                      shiftEndToShowOverrideForm={shiftEndToShowOverrideForm}
-                      onShowShiftSwapForm={!shiftSwapIdToShowForm ? this.handleShowShiftSwapForm : undefined}
-                      filters={filters}
-                    />
+                    {/* we need to render to allow Rotations show rotaion modal form */}
+                    <div style={{ display: scheduleView === ScheduleView.OneMonth ? 'none' : 'block' }}>
+                      <Rotations
+                        scheduleId={scheduleId}
+                        onCreate={this.refreshEventsAndClearPreview}
+                        onUpdate={this.refreshEventsAndClearPreview}
+                        onDelete={this.refreshEventsAndClearPreview}
+                        shiftIdToShowRotationForm={shiftIdToShowRotationForm}
+                        layerPriorityToShowRotationForm={layerPriorityToShowRotationForm}
+                        onShowRotationForm={this.handleShowRotationForm}
+                        onShowOverrideForm={this.handleShowOverridesForm}
+                        disabled={disabledRotationForm}
+                        filters={filters}
+                        onShowShiftSwapForm={!shiftSwapIdToShowForm ? this.handleShowShiftSwapForm : undefined}
+                        onSlotClick={shiftSwapIdToShowForm ? this.adjustShiftSwapForm : undefined}
+                      />
+                    </div>
+                    {/* we need to render to allow ScheduleOverrides show overrides modal form */}
+                    <div style={{ display: scheduleView === ScheduleView.OneMonth ? 'none' : 'block' }}>
```

Comment:
Can we apply class conditionally instead of using inline styles?

Reply:
yes, we can but seems it doesn't make a lot of sense here